src/schema/v2/user.ts

```diff
@@ -43,6 +43,23 @@ export const UserAdminNoteType = new GraphQLObjectType<any, ResolverContext>({
       type: new GraphQLNonNull(GraphQLString),
     },
     createdAt: date(({ created_at }) => created_at),
+    creator: {
+      description: "The user who created the note",
+      type: UserType,
```

Comment:
I see that in [Bidder type](https://github.com/artsy/metaphysics/blob/main/src/schema/v2/bidder.ts#L28) we use `UserField.type` when resolving the `user` tho I am not clear on what the difference is between using that and `UserType` that I am using here.